TaskModel.json

```diff
@@ -1801,6 +1801,79 @@
 			"plops": 100000,
 			"group": 1
 		},
+		{
+			"name": "Halle-Kasseler Bahn von %s nach %s",
+			"descriptions": [
+				"Bringe den Regionalexpress der Linie 8 von %s nach %s.",
+				"Bring die Fahrgäste in dem RE 8 pünktlich nach %2$s.",
+				"Fahre den RE störungsfrei nach %2$s."
+			],
+			"stations": [ "LH", "LRN", "LEL", "LWOR", "USG", "UBK", "UN", "UBD", "USOL", "UL" ],
+			"neededCapacity": [
+				{ "name": "passengers", "value": 0 }
+			],
+			"plops": 150000,
+			"group": 1
+		},
+		{
+			"name": "Halle-Kasseler Bahn von %s nach %s",
+			"descriptions": [
+				"Bringe den Regionalexpress der Linie 9 von %s nach %s.",
+				"Bring die Fahrgäste in dem RE 9 pünktlich nach %2$s.",
+				"Fahre den RE störungsfrei nach %2$s."
+			],
+			"stations": [ "LH", "LRN", "LEL", "LWOR", "USG", "UWAL", "UBEN", "UROS", "UBK", "UGSB", "UHRH", "UN", "UBD", "UL" , "UHT", "HEBG", "FWZN", "FHMD", "FKW"],
+			"neededCapacity": [
+				{ "name": "passengers", "value": 0 }
+			],
+			"plops": 175000,
+			"group": 1
+		},
+		{
+			"name": "RB 57 von %s nach %s",
+			"descriptions": [
+				"Bringe die RB über die Halle-Kasseler Bahn von %s nach %s.",
+				"Bring die Fahrgäste in der RB 57 pünktlich nach %2$s.",
+				"Fahre die RB störungsfrei nach %2$s."
+			],
+			"stations": [ "UN", "UBD", "UL" , "UHT"],
```

Comment:
Muss in ein Unterobjekt, schau dir das mal bei anderen Ausschreibungen mit mehreren Stationsstrings an!